### ISSUE TITLE
Python 3 support

### ISSUE THREAD

**Issue by aaroncohen73** [osuosl/pymesync#141]
Python 3 has been out for 8 years now, so we should support that.

**Comment by matthewrsj** [comment]
+1 on this. It would be great if it was cross-compatible.

**Comment by ElijahCaine** [comment]
Making things compatible with both versions of python is actually pretty painless in my experience. You essentially put in some try except statements:
```python
try:
    import python2_specific_deps as dep
except:
    try:
        import python3_specific_deps as dep
    else:
        print("Please import the dependencies with `pip install -r requirements.txt`")
        return
# do other imports
# use dep interchangeably.
```

Or you could get the version of python you are using with `sys.version_info` and do a normal `if, elif, else` conditional. Just thought I'd share how I've done it in the past.

**Comment by subnomo** [comment]
Closed by #157.
